aws/terminator/__init__.py

```diff
@@ -72,18 +64,18 @@ def process_instance(instance: 'Terminator', check: bool, force: bool = False) -
     return status
 
 
-def cleanup_test_account(stage: str, check: bool, force: bool, api_name: str, test_account_id: str, targets: typing.Optional[typing.List[str]] = None) -> None:
-    role = f'arn:aws:iam::{test_account_id}:role/{api_name}-test-{stage}'
-    credentials = assume_session(role, 'cleanup')
+def cleanup_resources(check: bool, force: bool, targets: typing.Optional[typing.List[str]] = None) -> None:
 
+    if targets:
+        targets = [t.lower() for t in targets]
     for terminator_type in sorted(get_concrete_subclasses(Terminator), key=lambda value: value.__name__):
-        if targets and terminator_type.__name__ not in targets:
+        if targets and terminator_type.__name__.lower() not in targets:
             continue
 
         # noinspection PyBroadException
         try:
             # noinspection PyUnresolvedReferences
-            instances = terminator_type.create(credentials)
+            instances = terminator_type.create()
```

Comment:
Why remove the session here?

Reply:
The current process needs to assume a Role from another AWS account, this means the current session in the lambda context is the one with the required permissions. With this new update the lambda session has the required perms, we no longer need to assume a role